### PR TITLE
Add name of invoked unknown function to error message

### PR DESCRIPTION
**Pull request by rick-nguyen**
https://github.com/microsoft/Power-Fx/issues/250

### CODE REVIEW

File: src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs [hunk 1]
```diff
@@ -4631,7 +4631,7 @@ public override bool PreVisit(CallNode node)
                 var overloads = LookupFunctions(funcNamespace, node.Head.Name.Value);
                 if (!overloads.Any())
                 {
-                    _txb.ErrorContainer.Error(node, TexlStrings.ErrUnknownFunction);
+                    _txb.ErrorContainer.Error(node, TexlStrings.ErrUnknownFunction, node.Head.Name.Value);
```
Comment (Shpakh-MSFT) [inline src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs:4634]: >node.Head.Name.Value [](http://example.com/codeflow?start=84&length=20)

If the funcNamespace is not global or empty then you may want to include it here.

**Comment by MikeStall** [comment]
Is there a test update? IF not, we should add a test. 
(PAClient will likely require a test update)

**Review by MikeStall — CHANGES_REQUESTED** [review]
:clock1:

**Comment by rick-nguyen** [comment]
Added a couple tests

**Review by MikeStall — APPROVED** [review]
:shipit:

**Pull request merged**